widgetry/src/widgets/panel.rs

```diff
@@ -180,10 +262,9 @@ impl Panel {
                 Polygon::rectangle(self.container_dims.width, self.container_dims.height)
                     .translate(top_left.x, top_left.y),
             );
+            g.unfork();
```

Comment:
This is *not* related to my change. And didn't *seem* to be causing problems as is... but it seemed wrong where it was. I assumed all `unfork` should have a corresponding `fork`.

I'm only like 1/2 sure I understand what `fork`ing is even about - I think it's about nesting coordinate systems, aking to [UIKit's#bounds](https://stackoverflow.com/questions/1210047/cocoa-whats-the-difference-between-the-frame-and-the-bounds/28917673#:~:text=The%20bounds%20of%20an%20UIView,superview%20it%20is%20contained%20within.).

Reply:
> I assumed all unfork should have a corresponding fork.

Correct, they should. Here must've been missing by mistake. `unfork` isn't strictly needed if there's a later explicit `fork`, which must've wound up being true here.

> I think it's about nesting coordinate systems

`fork` is about making the GPU do coordinate transformations. At first, it was just "screen-space" and "map-space". We want to do `map_to_screen` inside the vertex shader for things that can pan and zoom, but not for screen-space UI things. Later I started using it also to shift/compress map-space, like for the minimap. It's possible some of those uses could instead do the work on the CPU, calling `translate` and `scale` on the `GeomBatch`. Haven't tried that. I'd assume it'd be much slower initially for things like the minimap, which has a large amount of geometry.